frontend/lib/src/components/elements/ArrowVegaLiteChart/CustomTheme.tsx

```diff
@@ -89,13 +89,17 @@ export function applyStreamlitTheme(config: any, theme: EmotionTheme): any {
       titleFontWeight: theme.fontWeights.normal,
       titleFontStyle: "normal",
       titleColor: getGray70(theme),
+      // TODO(lukasmasuch): Change padding here to use a spacing
+      // based on our available spacings (-> 4px = 0.25rem)
       titlePadding: 5,
```

Comment:
Unfortunately, we cannot represent these values reliably in rem. We need to change that to use padding based on our spacing values.